cmd/krel/cmd/changelog.go

```diff
@@ -350,3 +361,44 @@ func lookupRemoteReleaseNotes(branch string) (string, error) {
 
 	return string(content), nil
 }
+
+func pushChanges(repo *git.Repo, branch string, tag semver.Version) error {
+	if !rootOpts.nomock {
+		logrus.Info("Using dry mode, which does not modify any remote content")
+		repo.SetDry()
```

Comment:
It seems a bit strange to me that the repo is set to "dryrun mode" only when this more or less random function is called. I'd assume, that all the components are constructed or setup as early with settings like that.

Reply:
Okay, I moved this part to happen directly after the `CloneOrOpenDefaultGitHubRepoSSH()`.